nncf/quantization/fake_quantize.py

```diff
@@ -122,11 +121,14 @@ def tune_range(
         fval = -left_border * s
         qval = fns.round(fval)
 
-    ra = fns.where(qval < level_high, qval / (qval - level_high) * right_border, left_border)
-    with warnings.catch_warnings():
-        # If `qval` is 0 `rb` will equal `right_border`, and we don't want to show an unnecessary division by 0 warning
-        warnings.simplefilter("ignore")
-        rb_then_result = (qval - level_high) / qval * left_border
+    # Avoid division by zero
+    qval_nonzero = fns.where(qval == 0, fns.ones_like(qval), qval)
```

Comment:
Please, check the performance of the function after your changes. @nikita-savelyevv, it looks like we discussed it already. Could you remember us the solution?

Reply:
As I remember ignoring the warning was the solution with the least impact.

Reply:
Perf measurment:
with warnings context manager: ~17.648 sec
without warning context manager but with: ~20.809 sec
```
qval_nonzero = fns.where(qval == 0, fns.ones_like(qval), qval)
qval_not_high = fns.where(qval - level_high == 0, fns.ones_like(qval), qval - level_high)
```

Reply:
> @kshpv you can use just `1.0` instead of `fns.ones_like(qval)`
> 
> ```
> qval_nonzero = fns.where(qval == 0, 1.0 , qval)
> qval_not_high = fns.where(qval - level_high == 0, 1.0 , qval - level_high)
> ```

I checked your proposed version and it has the same performance as with `fns.ones_like(qval)` :(

Reply:
I suggest to use implementation with the best performance.